app.rb

```diff
@@ -0,0 +1,112 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_books
+    return puts 'No books found' if @books.empty?
+
+    @books.each do |book|
+      puts "Title: \"#{book.title}\", Author: #{book.author}"
+    end
+  end
+
+  def list_people
+    return puts 'No people found' if @people.empty?
+
+    @people.each do |person|
+      puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+  end
```

Comment:
- [ ] Also consider looking at this bug:

![image](https://user-images.githubusercontent.com/77069011/232167008-f2bafcdf-9485-4303-926b-0d775cf1ff5b.png)

I think you're passing the arguments in the wrong order, please consider fixing that! 🤓 🔧